jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -458,6 +465,15 @@ public boolean apply(final Subscription input) {
         final BlockingStateJson blockingState = new BlockingStateJson(accountId, "CLOSE_ACCOUNT", "account-service", true, false, false, null, BlockingStateType.ACCOUNT, null);
         addBlockingState(blockingState, accountId, BlockingStateType.ACCOUNT, null, ImmutableList.<String>of(), createdBy, reason, comment, request);
 
+        if (removeFutureNotifications) {
+            final Long tenantRecordId = recordIdApi.getRecordId(callContext.getTenantId(), ObjectType.TENANT, callContext);
+            final Long accountRecordId = accountId == null ? null : recordIdApi.getRecordId(accountId, ObjectType.ACCOUNT, callContext);
+            for (final NotificationQueue notificationQueue : notificationQueueService.getNotificationQueues()) {
+                log.debug("Removing future notifications for queueName={}", notificationQueue.getFullQName());
+                notificationQueue.removeFutureNotificationsForSearchKeys(accountRecordId, tenantRecordId);
```

Comment:
From what i understood there is a limitation with the default batch size, hardcoded to `25` ?